<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWorker</class>
 <widget class="QWidget" name="AddWorker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加职工</string>
  </property>
  <widget class="QLabel" name="worker_degree_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>学位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_it_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加职工</string>
   </property>
  </widget>
  <widget class="QLabel" name="worker_title_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>职称</string>
   </property>
  </widget>
  <widget class="QLabel" name="worker_address_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkerNameEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="worker_date_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>入职日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="worker_id_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>工号</string>
   </property>
  </widget>
  <widget class="QLabel" name="worker_name_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="workerIDEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="worker_area_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>现有住房面积</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkerAddressEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="alertLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkerAreaEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="WorkerdateEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="DegreecomboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TitlecomboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>出生日期</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthdateEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
